src/python/pants/testutil/engine/util.py

```diff
@@ -201,3 +202,8 @@ def green(self, text):
 
   def red(self, text):
     return self._safe_color(text, red)
+
+
+class MockOptions:
+  def __init__(self, **values):
+    self.values = Mock(**values)
```

Comment:
It's fine to have this in the test file you need to use, but we don't want this in `testutil` yet because that is a public API. Generally, we're still in the discovery stage for V2 testing and have avoided trying to generalize too much until we have a more clear understanding of all the different use cases we need to support.